src/providers/ethereum/EthereumSwapProvider.js

```diff
@@ -95,15 +95,24 @@ export default class EthereumSwapProvider extends Provider {
     return ''
   }
 
-  async findInitiateSwapTransaction (value, recipientAddress, refundAddress, secretHash, expiration) {
+  doesTransactionMatchSwapParams (transaction, value, recipientAddress, refundAddress, secretHash, expiration) {
     const data = this.createSwapScript(recipientAddress, refundAddress, secretHash, expiration)
+    return transaction.input === data.toLowerCase() && transaction.value === value
```

Comment:
Should we standardize `.toLowerCase()` throughout Ethereum providers (seems to be an issue that comes up a lot, and we can't lowerCase on bitcoin side due to base58). Maybe future PR.

Reply:
Good point, added this as a `ensureAddressStandardFormat`